changelog.txt

```diff
@@ -14,6 +14,7 @@ Version 0.15
 Added
 +++++
 
+- Enable support for aggregation - workflows on groups of jobs (#464).
```

Comment:
Mentioning the most important new name in the API (`flow.aggregator`) is helpful.
```suggestion
- Add support for aggregation (operations acting on multiple jobs) via ``flow.aggregator`` (#464).
```